sdk/servicebus/azure-messaging-servicebus/pom.xml

```diff
@@ -47,7 +47,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core-amqp</artifactId>
-      <version>1.6.0-beta.1</version> <!-- {x-version-update;unreleased_com.azure:azure-core-amqp;dependency} -->
+      <version>1.7.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core-amqp;current} -->
```

Comment:
This should take unreleased_com depedency of core-amqp if SB needs active changes from the core-amqp source.

Reply:
Corrected to use unreleased_